graphql/schema.graphql

```diff
@@ -4,4 +4,12 @@ scalar Date @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\Date")
 "A datetime string with format `Y-m-d H:i:s`, e.g. `2018-05-23 13:43:32`."
 scalar DateTime @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\DateTime")
 
+type Mutation {
+    "Log in to a new session and get the user."
+    login(email: String!, password: String!): User!
```

Comment:
You **could** make the User return type nullable (null upon failed login), but that's a matter of taste.

Reply:
In case an Exception is thrown, field execution is aborted anyways and an error is produced.

It is generally better for clients to be able to rely on a result being there. I am particularly keen on it since we use https://graphql-code-generator.com/ coupled with strict TypeScript options to enforce proper handling of nullability.